packages/fetchai/agents/random_beacon_feed/aea-config.yaml

```diff
@@ -35,6 +35,6 @@ type: connection
 config:
   ledger_apis:
     fetchai:
-      address: https://rest-beaconworld.fetch.ai:443
```

Comment:
I'll check on this, but I believe that Stargateworld does not have a random beacon.